include/caf/leds.rst

```diff
@@ -18,19 +18,18 @@ To use the module, you must enable the following Kconfig options:
 * :option:`CONFIG_CAF_LEDS` - This option enables the LEDs module.
 * :option:`CONFIG_PWM` - This option enables Zephyr's :ref:`zephyr:pwm_api` driver, which is used for setting the LED color (that is, the brightness of the diodes).
   Setting this option requires you to define PWM ports, as described in the following section.
```

Comment:
The section was renamed, so the reference is unclear now.

Reply:
Section was renamed but the process is still in there. This is described in the following section at point 1.
The process of adding pwm led is described in the following section point 2.
Should I add information about points?